boards/nxp/s32z2xxdc2/board.yml

```diff
@@ -1,6 +1,6 @@
 board:
   name: s32z2xxdc2
-  full_name: X-S32Z27X-DC (DC2)
+  full_name: X-S32Z27X-DC2
```

Comment:
please revert this change, it's unnecessary. The existing name is how is currently documented in NXP documentation.

Reply:
@manuargue 

1. Could you provide a link to a board documentation and picture of the board?
They are absent in the Zephyr documentation.
2. Please clarify why used two different names for the board: s32z2xxdc2 and X-S32Z27X-DC (DC2)

Reply:
This is a preproduction part, documentation is not yet publicly available.

Reply:
> This is a preproduction part, documentation is not yet publicly available.

OK. Could you tell what is written on the circuit board? X-S32Z27X-DC2?

Reply:
in available documentation is written exactly as it currently in the code, no changes are required

Reply:
> in available documentation is written exactly as it's currently in the code, `X-S32Z27X-DC (DC2)`, no changes are required

Reverted. As the DC2 is not a revision.